packages/firestore/test/unit/local/remote_document_cache.test.ts

```diff
@@ -215,20 +216,20 @@ function eagerRemoteDocumentCacheTests(
     const doc1 = doc('docs/foo', 1, { foo: false, bar: 4 });
     const doc2 = doc('docs/bar', 2, { bar: 'yes', baz: 'also yes' });
 
-    await cache.addEntries([doc1]);
+    await cache.addEntries([doc1], version(1));
     const doc1Size = await cache.getSize();
     expect(doc1Size).to.equal(0);
 
-    await cache.addEntries([doc2]);
+    await cache.addEntries([doc2], version(2));
     const totalSize = await cache.getSize();
     expect(totalSize).to.equal(0);
 
-    await cache.removeEntry(doc2.key);
+    await cache.removeEntry(doc2.key, version(3));
```

Comment:
All of these removeEntry() versions are meaningless, right?  Which is why you can pass in version(3) here even though that is _not_ the version associated with doc2.

I think only addEntries() should require a version.

Reply:
"meaningless". Ouch.

But yes, they are.